dotnet/Face/FaceQuickstart.cs

```diff
@@ -53,821 +53,825 @@
-			Console.WriteLine();
-			DeletePersonGroup(client, sourcePersonGroup).Wait();
-			// </snippet_persongroup_delete>
-			// <snippet_target_persongroup_delete>
-			DeletePersonGroup(clientTarget, targetPersonGroup).Wait();
-			Console.WriteLine();
-			// </snippet_target_persongroup_delete>
-
-			Console.WriteLine("End of quickstart.");
-		}
-
-		// <snippet_auth>
-		/*
+    class Program
+    {
+        // Used for the Identify, Snapshot, and Delete examples.
+        // The same person group is used for both Person Group Operations and Snapshot Operations.
+        // <snippet_persongroup_declare>
+        static string sourcePersonGroup = null;
+        // </snippet_persongroup_declare>
+        // <snippet_snapshot_persongroup>
+        static string targetPersonGroup = null;
+        // </snippet_snapshot_persongroup>
+
+        // <snippet_image_url>
+        // Used for all examples.
+        // URL for the images.
+        const string IMAGE_BASE_URL = "https://csdx.blob.core.windows.net/resources/Face/Images/";
+        // </snippet_image_url>
+
+        static void Main(string[] args)
+        {
+            // Used in Authenticate and Snapshot examples. The client these help create is used by all examples.
+            // <snippet_mainvars>
+            // From your Face subscription in the Azure portal, get your subscription key and endpoint.
+            // Set your environment variables using the names below. Close and reopen your project for changes to take effect.
+            string SUBSCRIPTION_KEY = Environment.GetEnvironmentVariable("FACE_SUBSCRIPTION_KEY");
+            string ENDPOINT = Environment.GetEnvironmentVariable("FACE_ENDPOINT");
+            // </snippet_mainvars>
+
+            // <snippet_snapshot_vars>
+            // The Snapshot example needs its own 2nd client, since it uses two different regions.
+            string TARGET_SUBSCRIPTION_KEY = Environment.GetEnvironmentVariable("FACE_SUBSCRIPTION_KEY2");
+            string TARGET_ENDPOINT = Environment.GetEnvironmentVariable("FACE_ENDPOINT2");
+            // Grab your subscription ID, from any resource in Azure, from the Overview page (all resources have the same subscription ID). 
+            Guid AZURE_SUBSCRIPTION_ID = new Guid(Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID"));
+            // Target subscription ID. It will be the same as the source ID if created Face resources from the same
+            // subscription (but moving from region to region). If they are different subscriptions, add the other
+            // target ID here.
+            Guid TARGET_AZURE_SUBSCRIPTION_ID = new Guid(Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID"));
+            // </snippet_snapshot_vars>
+
+            // <snippet_detect_models>
+            // Used in the Detect Faces and Verify examples.
+            // Recognition model 3 is used for feature extraction, use 1 to simply recognize/detect a face. 
```

Comment:
The original comment seems to mistakenly refer to the detection model. I replaced it with this, copied from code comments in the SDK:
// Recognition model 3 was released in 2020 May.
// It is recommended since its overall accuracy is improved
// compared with models 1 and 2.